django/apps/config.py

```diff
@@ -290,3 +290,14 @@ def ready(self):
         """
         Override this method in subclasses to run code when Django starts.
         """
+
+    @property
```

Comment:
I think the only case where it will change at runtime is when running the Django tests. I'll see if making this a cached property has any impact with those.